src/include/btree_inline.h

```diff
@@ -1737,6 +1737,7 @@ __wt_leaf_page_can_split(WT_SESSION_IMPL *session, WT_PAGE *page)
     WT_INSERT_HEAD *ins_head;
     size_t size;
```

Comment:
Let's define all the variables with the same type together.
```suggestion
    size_t mem_split_threshold, size;
```